Cargo.toml

```diff
@@ -1,19 +1,21 @@
 [package]
-
-name = "stomp"
-version = "0.11.0"
-authors = [ "zslayton <zack.slayton@gmail.com" ]
-repository = "https://github.com/zslayton/stomp-rs"
-readme = "README.md"
+authors = ["zslayton <zack.slayton@gmail.com"]
 description = "A full STOMP 1.2 client implementation. Allows programs to interact with message queueing services like ActiveMQ and RabbitMQ."
-keywords = [ "stomp", "mq", "activemq", "rabbitmq", "queue" ] 
+keywords = ["stomp", "mq", "activemq", "rabbitmq", "queue"]
 license = "MIT"
-
-[lib]
 name = "stomp"
+readme = "README.md"
+repository = "https://github.com/zslayton/stomp-rs"
+version = "0.13.1"
```

Comment:
Since the current (broken) version of `stomp-rs` is v0.11.0, could you change this to v0.12.0?

Reply:
Fixed.